CollAction/Frontend/app/global/styles/footer.scss

```diff
@@ -117,5 +89,76 @@
             text-decoration: underline;
         }
     }
+
+
+    //Newsletter signup
+    .newsletter-signup {
+        position: relative;
+    }
+
+    .newsletter-signup__container {
```

Comment:
Bit of a weird naming convention. Why two underscores?

Reply:
It's the block-element-modifier (BEM) convention: https://csswizardry.com/2013/01/mindbemding-getting-your-head-round-bem-syntax/. In this case newsletter-signup is the block and container the element. 

We used this at my previous employer for all websites. It may look weird in the beginning but works quite well. I woud like to introduce this convention in the whole frontend later on. But we can discuss that.